src/Loggers/SQL/DoctrineSQLLogger.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace LaravelDoctrine\ORM\Loggers\SQL;
+
+use Doctrine\DBAL\Logging\SQLLogger;
+use Doctrine\ORM\Configuration;
+use Doctrine\ORM\EntityManagerInterface;
+use Illuminate\Database\Events\QueryExecuted;
+
+class DoctrineSQLLogger implements SQLLogger
+{
+    /** @var float|null */
+    public $start = null;
+
+    /** @var string|null */
+    public $query = null;
+
+    /** @var array|null */
+    public $params = null;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    public $em;
+    /**
+     * @var Configuration
+     */
+    public $configuration;
+
+    public function __construct(EntityManagerInterface $em, Configuration $configuration)
+    {
+        $this->em            = $em;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function startQuery($sql, ?array $params = null, ?array $types = null)
+    {
+        $this->start  = microtime(true);
+        $this->query  = $sql;
+        $this->params = $params;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function stopQuery()
+    {
+        $executionTime = microtime(true) - $this->start;
+        // TODO get connection name
+        // this class is Laravel connection wrapper (QueryExecuted needs getName())
+        $object                    = new class {
```

Comment:
This should actually be a `\Illuminate\Database\Connection` instance. Though implementing that seems like a pretty huge task.

Reply:
The laravel-telescope QueryWatcher uses the ->connection->prepareBindings on this event: https://github.com/laravel/telescope/blob/e39423d3ee18898eeb1e0adf2fbe7b74a4cc4996/src/Watchers/QueryWatcher.php

Reply:
We have 2 options i guess. 

1. extend my anonymous object (add prepareBindings)
2. try to implement [ConnectionInterface](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Database/ConnectionInterface.php) with empty methods. This methots could be extended in fugure if it will be nessesery.

I will try the 2nd option. Create a wrapper for DoctrineConnection with Laravel ConnectionInterface.

Reply:
@eigan I've maybe found a way. Please check my last commit https://github.com/laravel-doctrine/orm/pull/501/commits/18ac1bef240fe6c97bd3a4c0883be467b54c0fab

Reply:
Cool. Could you make some tests for this too?

Reply:
> Cool. Could you make some tests for this too?

I've check Sentry support and it works.

Reply:
I was thinking about PHPUnit test.

Reply:
I've added few tests.